src/golang/cmd/server/request/payload.go

```diff
@@ -55,7 +55,10 @@ func ExtractHttpPayload(contentType, fileName string, isFile bool, r *http.Reque
 		} else {
 			// If we reach here, it means the value is sent as `String` instead of `File`,
 			// so we use `FormValue` to extract the payload and return its byte form.
-			// The caller can later parse the returned value to string.
+			// The caller can later parse the returned value to string. By default, the
```

Comment:
A reference to how I derive this kind of behavior is from https://cs.opensource.google/go/go/+/refs/tags/go1.20.2:src/net/http/request.go;l=1325
https://cs.opensource.google/go/go/+/refs/tags/go1.20.2:src/mime/multipart/formdata.go;l=31

Essentially, the behavior of `FormValue` is that:
1. request.body size < 42MB, everything is good 
2. request.body size > 42MB, Reading from request is not successful->empty string is returned-> no error is raised-> `json.unmarshal` on empty string -> error.

I am not sure the necessity of taking care of second case, since I am not sure if we have any instance where size of dag is larger than the default 42MB besides `localData`.